### ISSUE TITLE
overview: Make usage bars the same width

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#16506]
The overview page can have different sizes for the progress bars. They should be aligned on both sides, with the same width.

**Comment by Vishal-K-988** [comment]
Hey, this is my first time contributing to this project.  Can you please guide me on how to start working on this Issue/project?

**Comment by martinpitt** [comment]
@Vishal-K-988 : I suggest the following:

 1. Check out the git repo, learn how to build it and run it. Change a string in the code somewhere and make sure you can see the change in the browser. See [HACKING.md](https://github.com/cockpit-project/cockpit/blob/main/HACKING.md) for details.
 2. 
 3. Build a test VM and run a single test on your machine. Run it again with showing/watching the browser (normally it runs headless). See [test/README.md](https://github.com/cockpit-project/cockpit/blob/main/test/README.md) for details
 
Ask in #cockpit if you run into any trouble, we are happy to help. Getting to this point can take up to a day depending on your familiarity with Linux tools and development. After that, you are ready for taking on an issue like this one, assuming you have some basic familiarity with HTML, CSS, and JS.

Enjoy, and thanks for your interest!

**Comment by garrett** [comment]
Note: This is actually quite tricky and probably really needs to use subgrid to properly work. Subgrid has been supported for several years in Firefox and has been in Safari for almost half a year. However, it's not supported in Chromium, and there's still no sign of adoption there (sadly).

https://caniuse.com/css-subgrid

Subgrid is the obvious solution. There is no other solution. So we cannot fix this for the dominant browser engine. That's why I haven't worked on this yet.

There's no other solution that would work.

Hardcoding the widths won't work either, as languages and browser sizes will change the widths of the strings and the amount of space used. (And hardcoding in general is not a good idea anyway.)

The only other solution we have is to rewrite this to not use PatternFly widgets at the top level, so we could use a grid form of layout (either a grid or even a table, but not anything else) to encapsulate the two rows and each item within the row at the same level in HTML.

Between these two solutions, you can see why this stalled. It's still an issue, and I've been hopeful that the Chrome team would finally implement subgrid... but like anything else, Chrome is quick to implement its own non-standard features without a spec and slow to implement an actual agreed upon standards-based spec. (This is one of the many problems of Google controlling the web as they do in the de facto way of owning Chromium.)

In other words, this is not really a good first issue and much trickier than it seems.
